src/framework/components/render/component.js

```diff
@@ -934,23 +938,20 @@ class RenderComponent extends Component {
     }
 
     _cloneSkinInstances() {
-
-        if (this._meshInstances.length && this._rootBone.entity instanceof GraphNode) {
-
+        if (this._meshInstances.length && this._rootBone instanceof GraphNode) {
             for (let i = 0; i < this._meshInstances.length; i++) {
                 const meshInstance = this._meshInstances[i];
                 const mesh = meshInstance.mesh;
 
                 // if skinned but does not have instance created yet
                 if (mesh.skin && !meshInstance.skinInstance) {
-                    meshInstance.skinInstance = SkinInstanceCache.createCachedSkinInstance(mesh.skin, this._rootBone.entity, this.entity);
+                    meshInstance.skinInstance = SkinInstanceCache.createCachedSkinInstance(mesh.skin, this._rootBone, this.entity);
                 }
             }
         }
     }
 
     _cloneMeshes(meshes) {
-
```

Comment:
no need to remove empty lines in this PR :)

Reply:
Couldn't resist, as most places in this file do not have these unnecessary empty lines in the same context.
Do you want me to add them back?

Reply:
Personally, I don't mind removing them. Our coding standards don't state you should add a blank line after the first line of a function. I we want to introduce that rule, ok, we can debate it - but I'm more keen that we enforce consistency across the whole codebase.